docs/tutorials/hello.rst

```diff
@@ -1,72 +1,75 @@
+używamy polecenia ``scroll`` (ang. przewiń). Polecenie ``scroll``
+musi wiedzieć jakie znaki pokazać na wyświetlaczu. Wpisujemy je ujęte
+w cudzysłów (``"``) i zamknięte w nawiasy (``(`` i ``)``). W programowaniu nazywamy
+to *argumentami* (ang. arguments). Zatem ``display.scroll("Hello, world!")``
+oznacza, po angielsku, "chcę użyć wyświetlacza by pokazać przesuwający się
+tekst 'Hello, world!'". Jeśli metoda nie potrzebuje żadnych argumentów,
+musimy to jasno określić używając pustych nawiasów: ``()``.
+
+Skopiuj powyższy kod do swojego edytora i zaprogramuj go (ang. flash) na
+urządzeniu. Czy domyślasz się jak zmienić wyświetlany tekst? Czy możesz
+go zmienić tak, by przywitał ciebie? Na przykład, chciałbym by tekst brzmiał
+"Witaj, Andrzeju!". Mała podpowiedź: musisz zmienić argument metody ``scroll``.
 
 .. warning::
 
-    It may not work. :-)
-
-    This is where things get fun and MicroPython tries to be helpful. If
-    it encounters an error it will scroll a helpful message on the micro:bit's
-    display. If it can, it will tell you the line number for where the error
-    can be found.
-
-    Python expects you to type **EXACTLY** the right thing. So, for instance,
-    ``Microbit``, ``microbit`` and ``microBit`` are all different things to
-    Python. If MicroPython complains about a ``NameError`` it's probably
-    because you've typed something inaccurately. It's like the difference
-    between referring to "Nicholas" and "Nicolas". They're two different people
-    but their names look very similar.
-
-    If MicroPython complains about a ``SyntaxError`` you've simply typed code
-    in a way that MicroPython can't understand. Check you're not missing any
-    special characters like ``"`` or ``:``. It's like putting. a full stop in
-    the middle of a sentence. It's hard to understand exactly what you mean.
-
-    Your microbit may stop responding: you cannot flash new code to it or
-    enter commands into the REPL. If this happens, try power cycling it. That
-    is, unplug the USB cable (and battery cable if it's connected), then plug
-    the cable back in again. You may also need to quit and re-start your code
-    editor application.
+    To może nie działać :-)
+    
+    Tutaj zaczyna się prawdziwa zabawa, a MicroPython stara się być naprawdę
+    pomocny. Jeśli napotka błąd, to na wyświetlaczu pokaże komunikat błędu.
+    W miarę możliwości, pokaże również numer linii kodu, która zawiera błąd.
+
+    Python oczekuje, że wprowadzisz **BEZBŁĘDNY** kod. Na przykład, ``Microbit``,
+    ``microbit`` i ``microBit`` są przez Pythona traktowane jak trzy osobne
+    rzeczy. Jeśli w treści błędu zobaczysz ``NameError``, oznacza to że
+    prawdopodobnie wpisana nazwa została podana niedokładnie. To jak różnica
+    między "Pawłem" i "Gawłem" - dwa różne imiona, choć brzmią i wyglądają
+    podobnie.
+    
+    Natomiast jeśli w treści błędu zobaczysz ``SyntaxError``, oznacza to po
```

Comment:
Czy warto wspomnieć, że to znaczy "błąd składni"?

Reply:
Dodane.